src/plugins/system/jtfavorites/install/sql/mysql/install.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS `#__jtfavorites` (
+  `extension_id` int(11) NOT NULL,
+  `user_id` int(11) NOT NULL,
+  `name` varchar(100),
+  PRIMARY KEY (`extension_id`, `user_id`),
```

Comment:
Bitte das Komma am Ende der Zeile entfernen, da der SQL-Befehl fehlschlägt.
```suggestion
  PRIMARY KEY (`extension_id`, `user_id`)
```